hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/ResourceManagerRest.md

```diff
@@ -1753,6 +1753,70 @@ Response Body:
 </node>
 ```
 
+Cluster Node Update APIs
+------------------------
+
+You can update the resources reported from a Nodemanager using this API. To perform the node update, you must have authentication set up (static web user won't work) and have permissions to call the ResourceManager Admin commands as that user. The response is empty with a 202 or 200 status 
+
+### URI
+
+      * http://<rm http address:port>/ws/v1/cluster/node/{nodeId}/vcores
+      * (This url is case sensitive)
+
+### HTTP Operations Supported
+
+      * PUT
+
+### Query Parameters Supported
+
+      None
+
+### Elements of the NodeState object
```

Comment:
When reading this, I am not sure what NodeState is. Do you mean NodeCapacity?  Can you provide a link and/or make clear that the parameter is a json make it clear?